assets/scss/_custom.scss

```diff
@@ -416,6 +416,9 @@ body.td-home .deprecation-warning, body.td-blog .deprecation-warning, body.td-do
     border-radius: 3px;
 }
 
+.deprecation-warning p:last-child {
+  margin-bottom: 0;
+}
```

Comment:
I think this, as proposed, could be more fragile than intended. The existing margins for deprecation warnings look OK to me. See for example https://v1-21.docs.kubernetes.io/docs/home/ (which includes a warning) - are any of those margins / paddings too large on typical desktop viewports?

Going into specifics, the selector I'd use here would look at whether the `<p>` is a [general sibling](https://css-tricks.com/almanac/selectors/g/general-sibling/) of `<h3>` (or possibly: of any heading element). It's reasonable to have different margins when there is / isn't a heading.

I'm also OK with a more responsive approach that sets the margin based on the viewport dimensions, or similar.

Reply:
good call, switched to p:only-child